projects/angular/src/utils/focus/key-focus/util.ts

```diff
@@ -39,3 +39,9 @@ export function preventArrowKeyScroll(event: KeyboardEvent) {
     event.preventDefault();
   }
 }
+
+export function isKeyALetter(event: KeyboardEvent) {
+  const char = event.key;
+  // only letter characters differ when they switch between lowercase and uppercase.
+  return char.toLowerCase() !== char.toUpperCase() || (char >= '0' && char <= '9');
```

Comment:
this is including numbers? The name isn't very descriptive in that case

Reply:
You are right. I will find a better name and update it.